mapper/tedge_mapper/Cargo.toml

```diff
@@ -28,7 +28,7 @@ env_logger = "0.8"
 flockfile = {path = "../../common/flockfile" }
 log = "0.4"
 mqtt_client = {path = "../../common/mqtt_client" }
-tokio = { version = "1.1", features = ["full"] }
+tokio = { version = "1.6", features = ["rt", "sync", "time"] }
```

Comment:
This is good to just use the features needed. Does it really import only the features that are declared here into the binary/lib?. If so, we should find a way to check just required/dependancy features to import into the binary, this way we can save some memory.

Reply:
I think, but I might be wrong, that this has impacts on the cache size, build time and possible build issues but not on the target size.

Reply:
This is correct, specifying actually used features should decrease size of final binary as some packages may not be included. It is usually documented in crates docs. Eg `tokio`: https://docs.rs/tokio/1.6.1/tokio/#feature-flags